common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -275,5 +278,78 @@ object CuratedRecipe {
     })
   }
 
+  def toAtom(r: Recipe, cr: CuratedRecipe): Atom = {
+
+    val contentChangeDetails = ContentChangeDetails(
+      created = Some(
+        ChangeRecord(
+          date = r.publicationDate.toInstant.toEpochMilli,
+          user = Some(
+            User(email = "off-platform@guardian.co.uk")
+          )
+        )
+      ),
+      published = Some(
+        ChangeRecord(
+          date = r.publicationDate.toInstant.toEpochMilli,
+          user = Some(
+            User(email = "off-platform@guardian.co.uk")
+          )
+        )
+      ),
+      revision = 1L
+    )
+
+    val recipeAtom = atom.RecipeAtom(
+      title = cr.title,
+      tags = atom.Tags(
```

Comment:
1. We are no longer collecting `dietary`, it should not be a part of the model. In fact I am about to remove `dietary` and `holidays` from the main code/model. 
2. What is `celebration` ?
3. Did you mean to use `cuisines` 4 times ?